main.py

```diff
@@ -157,7 +163,13 @@ def handle_pr_review_comment(*, token, owner, repo_name, pull_request, comment):
     except Exception as e:
         logger.error(f"An error occurred: {str(e)}")
         import traceback
-        logger.error(f"Full traceback:\n{traceback.format_exc()}")
+        error_traceback = traceback.format_exc()
+        logger.error(f"Full traceback:\n{error_traceback}")
+        
+        # Post a comment about the error
+        error_comment = f"An error occurred while processing this PR review comment:\n\n```\n{str(e)}\n\n{error_traceback}\n```"
+        github_api.create_pr_comment(token, owner, repo_name, pull_request['number'], error_comment)
```

Comment:
This should be a reply to the PR review comment, not just a comment on the PR.

Reply:
@Aiderbot this should be a reply to the PR review comment, not just a comment on the PR.